src/solver/activ/bwd_0.cpp

```diff
@@ -46,9 +46,9 @@ bool ActivBwdSolver0::IsApplicable(const ExecutionContext&,
         return false;
 
     const auto& xDesc  = problem.GetXDesc();
-    const auto& yDesc  = problem.GetXDesc();
+    const auto& yDesc  = problem.GetYDesc();
```

Comment:
@DrizztDoUrden Please notice this typo. I am wondering why activation tests do not catch this (shall we add a test case?)

Reply:
Let's merge this PR first, I totally agree that we should add a test case for it.